File: 151001/Kononchuk/REST/src/REST/Models/Entities/Editor.cs
namespace REST.Models.Entities;

public class Editor
{
    public long Id { get; set; }
    public string Login { get; set; }
    public string Password { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }

    public List<Issue> Issues { get; set; } = new();
}

File: 1510031/Shamenov/project/lab1/src/main/java/by/bsuir/dc/lab1/entities/Editor.java
package by.bsuir.dc.lab1.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigInteger;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Editor {

    private BigInteger id;

    private String login;

    private String password;

    private String firstname;

    private String lastname;
}
